test_size.py

```diff
@@ -0,0 +1,29 @@
+from matryoshka_tester.parse_data import containers
+
+
+def test_size(host, container_runtime):
+    all_images = filter(
+        None,
+        host.run_expect(
+            [0],
+            f"{container_runtime.runner_binary} images "
+            + '--format "{{ .Repository }}:{{ .Tag }} {{ .Size }}"',
```

Comment:
there is a filter argument to the docker images command that will make the get_container_by_image_name disappear.

Reply:
Yes, that's a good idea. I've changed it.